cmd/juju/main.go

```diff
@@ -15,10 +13,6 @@ import (
 	_ "github.com/juju/juju/internal/provider/all" // Import the providers.
 )
 
-func init() {
```

Comment:
I think I removed it because i was checking on init stuff and rand.Seed is deprecated since 1.20:

https://pkg.go.dev/math/rand#Seed

So the purpose was "it was once useful". Since we have updated go version this should be removed.